discovery-provider/src/tasks/index_solana_plays.py

```diff
@@ -408,6 +500,7 @@ def process_solana_plays(solana_client_manager: SolanaClientManager, redis):
             page_count={page_count}"
         )
         page_count = page_count + 1
+        time.sleep(2) # sleep to not overwhelm rpc pool
```

Comment:
wondering why some timeout/limit vars like this are declared as constants up top and some aren't